src/renderer/components/drawer/drawer.tsx

```diff
@@ -125,9 +137,21 @@ export class Drawer extends React.Component<DrawerProps> {
     if (open) onClose();
   };
 
+  copyK8sObjName = () => {
+    const { title } = this.props;
+    const k8sObjName = title.toString().split(': ')[1];
+    clipboard.writeText(k8sObjName);
+    this.setState({copyIcon: "done", isCopied: true});
+    setTimeout(() => {
+      this.setState({copyIcon: "content_copy", isCopied: false});
+    }, 400);
```

Comment:
Please increase timeout to 3 or 4 seconds. Now it changes too fast:
![fast copy button](https://user-images.githubusercontent.com/9607060/129712133-f4a7062d-4d7d-4637-a917-61b1f562d1bd.gif)